nemo/backends/pytorch/actions.py

```diff
@@ -25,9 +27,8 @@
 
 # these imports will happen on as-needed basis
 amp = None
-convert_syncbn = None
-create_syncbn_process_group = None
-DDP = None
+# convert_syncbn = None
```

Comment:
@redoctopus are you training base models on multi-node setup with sync batch norm?

Reply:
Not currently.

Reply:
Sync batchnorm is still supported.